locator/token_metadata.cc

```diff
@@ -367,7 +366,8 @@ future<token_metadata_impl> token_metadata_impl::clone_only_token_map(bool clone
         return do_for_each(_token_to_endpoint_map, [&ret] (const auto& p) {
             ret._token_to_endpoint_map.emplace(p);
         }).then([this, &ret] {
-            ret._nr_normal_token_owners = _nr_normal_token_owners;
+            ret._normal_token_owners = _normal_token_owners;
+        }).then([this, &ret] {
```

Comment:
Any reason to add the then here?

Reply:
The idea was to allow yielding, but it turns out we don't check `need_preempt` in .then when the future is ready.
so the reason is moot.

Reply:
I see.

Reply:
So why do it?

Reply:
In hindsight it's unneeded. I can remove it in v2.
(No CI ran apparently so pushing a new version should trigger one)